tutorials/Linear Combination of Unitaries/LCU Tutorial .ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
<p>Please add a high-level figure / GIF  of the entire algorithm in this section in order to convey the high-level idea</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>